cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/ProgramRuntimeService.java

```diff
@@ -95,4 +97,5 @@
    * returns True if a program is running as defined by the predicate.
    */
   boolean checkAnyRunning(Predicate<Id.Program> predicate, ProgramType... types);
+  Set<Id.Program> getAnyRunning(Predicate<Id.Program> predicate, ProgramType... types);
```

Comment:
The doc needs to change too with the change in behavior. Though, I think this will move somewhere else according to terence's comment earlier. Change the function doc wherever you change the public apis.